app/tree/app/models/graphql/GraphQlSchemaDefinition.scala

```diff
@@ -389,6 +465,23 @@ object GraphQlSchemaDefinition extends AbstractGraphQlSchemaDefinition{
     Field("getBlocksByNta", ListType(BlockType), arguments= NtaArgument :: Nil, resolve = (ctx) => ctx.ctx.store.getBlocksByNta(nta = ctx.args.arg("nta"))),
```

Comment:
Convention is to not prefix GraphQL fields with a verb, so the query is { blocksByNta { ... } }

Reply:
I feel like this is too many root queries. Can you make it more graph-like? You can do things like blocks { hierarchy(inputs) geometries(inputs) }.